website/docs/r/cis_certificate_upload.html.markdown

```diff
@@ -0,0 +1,68 @@
+---
+layout: "ibm"
+page_title: "IBM: ibm_cis_certificate_upload"
+sidebar_current: "docs-ibm-resource-cis-certificate-upload"
+description: |-
+  Provides a IBM CIS Certificate Upload resource.
+---
+
+# ibm_cis_certificate_upload
+
+Provides a IBM CIS Certificate upload resource. This resource is associated with an IBM Cloud Internet Services instance and a CIS Domain resource. It allows to upload, update, delete certificates of a domain of a CIS instance
+
+## Example Usage
+
+```hcl
+# Upload a certificate for a domain
+
+resource "ibm_cis_certificate_upload" "cert" {
+    cis_id        = data.ibm_cis.cis.id
+    domain_id     = data.ibm_cis_domain.cis_domain.domain_id
+    certificate   = "xxxxx"
+		private_key   = "xxxxx
+		bundle_method = "ubiquitous"
+		priority      = 20
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+- `cis_id` - (Required,string) The ID of the CIS service instance
+- `domain_id` - (Required,string) The ID of the domain to add the Certificate Upload rule.
+- `certificate` - (Required,string) The intermediate(s) certificate key.
+- `private_key` - (Required,string) The certificate private key.
+- `bundle_method` - (Optional,string) The certificate bundle method. The valid values are: `ubiquitous`, `optimal`, `force`.
+- `priority` - (Optional,string) The order/priority in which the certificate is used in a request.
```

Comment:
priority is int in schema.. type mismatch

Reply:
added the priority constant